File: JanggiGame/SocketClient/Messages/MessageAction.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace SocketClient.Messages
{
    [JsonConverter(typeof(StringEnumConverter))]
    public enum MessageAction
    {
        NEW_GAME,
        GAME_STARTED,
        SETUP_COMPLETED,
        SETUP_CONFIRMED,
        GET_GAME_STATUS,
        GAME_STATUS,
        GET_PIECE_DESTINATIONS,
        PIECE_DESTINATIONS,
        MOVE_COMPLETED,
        MOVE_CONFIRMED,
        END_GAME,
        GAME_OVER,
        DEFAULT 
    }
}

File: JanggiGame/SocketServer/messages/message_action.py
from enum import auto, Enum


class MessageAction(Enum):
    NEW_GAME = auto()
    GAME_STARTED = auto()
    SETUP_COMPLETED = auto()
    SETUP_CONFIRMED = auto()
    GET_GAME_STATUS = auto()
    GAME_STATUS = auto()
    GET_PIECE_DESTINATIONS = auto()
    PIECE_DESTINATIONS = auto()
    MOVE_COMPLETED = auto()
    MOVE_CONFIRMED = auto()
    END_GAME = auto()
    GAME_OVER = auto()
    DEFAULT = auto()
